scripts/databricks_install.py

```diff
@@ -56,7 +56,9 @@
     "pydocumentdb==2.3.3",
 ]
 
-MMLSPARK_INFO = {"maven": {"coordinates": "Azure:mmlspark:0.16"}}
+MMLSPARK_INFO = {"maven": {"coordinates": "com.microsoft.ml.spark:mmlspark_2.11:0.16.dev8+2.g6a5318b",
```

Comment:
there is a problem with the new MMLSpark version and the current databricks installer. Earlier, @anargyri and I were trying to install it and we were getting errors. We need to take a look

Reply:
this is solved in the next commit